src/routes/articles.js

```diff
@@ -56,18 +59,15 @@ articlesRouter
   .get(slugChecker, Comment.getAll)
   .post(verifyToken, slugChecker, Comment.create);
 
-articlesRouter
-  .route(`${articlesBaseEndpoint}/:slug/comments/:commentId`)
-  .get([slugChecker, Comment.getOne]);
-
 articlesRouter
   .route(`${articlesBaseEndpoint}/:slug/comments`)
   .get(slugChecker, Comment.getAll)
   .post(verifyToken, slugChecker, Comment.create);
 
 articlesRouter
   .route(`${articlesBaseEndpoint}/:slug/comments/:commentId`)
-  .get(slugChecker, Comment.getOne);
+  // .get(slugChecker, Comment.getOne);
```

Comment:
Please remove comment

Reply:
Nice observation @marcdomain. Will do that